src/Controls/src/Core/Application.cs

```diff
@@ -167,8 +167,11 @@ public AppTheme UserAppTheme
 				TriggerThemeChangedActual(new AppThemeChangedEventArgs(value));
 			}
 		}
+
+		public AppTheme PlatformAppTheme => AppInfo.RequestedTheme;
```

Comment:
Added a new property to get the OS theme so as not to break existing code in a subtle way. I know @PureWeen you suggested not doing this, but I just wanted to mention the potential breaks and if you thought it was worth it?